ip-handling/draft-ietf-rtcweb-ip-handling.xml

```diff
@@ -123,46 +123,28 @@
-          requires no data exchange; it simply provides a mechanism for applications to retrieve the desired information from the kernel routing table.</t>
-          <t>Gathering all possible candidates SHOULD only be performed when
-          some form of user consent has been provided; this thwarts the typical drive-by enumeration attacks. The details of this consent are left to the implementation; one potential mechanism is to key this off getUserMedia consent. The getUserMedia suggestion takes into account that the user has provided some consent to the application already; that when doing so the user typically wants to engage in a conversational session, which benefits most from an optimal network path, and lastly, the fact that the underlying issue is complex and difficult to explain, making explicit consent for enumeration troublesome.</t>
-          <t>Determining whether a web proxy is in use is a complex process, as the answer can depend on the exact site or address being contacted. Furthermore, web proxies that support UDP are not widely deployed today. As a result, when WebRTC is
-          made to go through a proxy, it typically must use TCP, either ICE-TCP
-          <xref target="RFC6544"/> or TURN-over-TCP <xref target="RFC5766"/>. Naturally, this has attendant costs on media quality and also proxy performance.</t>
-          <t>RETURN <xref target="I-D.ietf-rtcweb-return"/> is a new proposal for explicit proxying of WebRTC media traffic. When RETURN proxies are
-          deployed, media and STUN checks will go through the proxy, but without the performance issues associated with sending through a typical web proxy.</t>
-        </list>
+      <t>We define four modes of WebRTC behavior, reflecting different privacy/media tradeoffs:
+      <list style="format Mode %d:">
+        <t>Enumerate all addresses: WebRTC will bind to all interfaces individually and use them all to attempt communication with STUN servers, TURN servers, or peers.  This will converge on the best media path, and is ideal when media performance is the highest priority, but it discloses the most information.</t>
+        <t>Enumerate all addresses: WebRTC will bind to all interfaces individually and use them all to attempt communication with STUN servers, TURN servers, or peers.  This will converge on the best media path, and is ideal when media performance is the highest priority, but it discloses the most information.</t>
```

Comment:
Looks like this got munged somehow. 

This patch is hard to compare because of the rearrangement. I am going to make a new PR without the rearrangement and then we can look at rerarranging post-merge.